lcls1_producers/prod_config_mec.py

```diff
@@ -0,0 +1,163 @@
+import numpy as np
+
+# These lists are needed, do not delete them
+# If no detector in a given category, leave the corresponding
+# list empty.
+detnames = ['alvium_spot',
+            'alvium_nf_ff',
+            'Zyla_xray_east',
+            'alvium_tt',
+            'epix100a_2_bxrts',
+            'epix100a_1_xtcs',
+            'Epix10kaQuad2',
+            'Epix10kaQuad3']
+
+# 1) REGIONS OF INTEREST
+def getROIs(run):
+    """ Set parameter for ROI analysis. Set writeArea to True to write the full ROI in the h5 file.
+    See roi_rebin.py for more info
+    """
+    if isinstance(run,str):
+        run=int(run)
+    ret_dict = {}
+    if run<100:
+        roi_dict = {}
+        roi_dict['name'] = 'ROI_0'
+        roi_dict['writeArea'] = True
+        roi_dict['thresADU'] = None
+        roi_dict['ROI'] = [[157,487], [294,598]]
+        ret_dict['Zyla_xray_east'] = [ roi_dict ]
+    elif run<250:
+        rz_dict = {'name':'ROI',
+                   'writeArea': True,
+                   'thresADU': None,
+                   'ROI':[[157,487], [294,598]]}
+        ret_dict['Zyla_xray_east'] = [ rz_dict ]
+
+        re11_dict = {'name':'ROI',
+                    'writeArea': True,
+                    'thresADU': None,
+                    'ROI':[[357,707], [0,768]],
+                   'pj': [{'name':'pj1', 'axis':0, 'thresADU': 0.0 }]}
+        re12_dict = {'name':'ROI_t',
+                    'writeArea': True,
+                    'thresADU': 7.0,
+                    'ROI':[[357,707], [0,768]],
+                   'pj': [{'name':'pjt', 'axis':0, 'thresADU': 7.0 }]}
+        ret_dict['epix100a_1_xtcs'] = [ re11_dict, re12_dict ]
+
+        re21_dict = {'name':'ROI1',
+                     'writeArea': True,
+                     'thresADU': None,
+                    'ROI':[[0,707], [360,370]],
+                    'pj': [{'name':'pj21', 'axis':1, 'thresADU': 0.0 }]}
+        re22_dict = {'name':'ROI2',
+                     'writeArea': True,
+                     'thresADU': None,
+                     'ROI':[[0,707], [310,320]],
+                     'pj': [{'name':'pj22', 'axis':1, 'thresADU': 0.0 }]}
+        re2t_dict = {'name':'ROIt',
+                     'writeArea': True,
+                     'thresADU': 1,
+                     'ROI':[[0,707], [360,370]],
+                     'pj': [{'name':'pj2t', 'axis':1, 'thresADU': 1.0 }]}
+        ret_dict['epix100a_2_bxrts'] = [ re21_dict, re22_dict, re2t_dict ]
+        
+    return ret_dict
```

Comment:
Should I clean up this file? Somehow having run numbers in the template feels wrong... It should all only have the "if run>0:" part.
What do you think?